packages/x-components/src/x-modules/queries-preview/store/module.ts

```diff
@@ -12,9 +12,14 @@ import { loadedQueriesPreview } from './getters/loaded-queries-preview.getter';
 export const queriesPreviewXStoreModule: QueriesPreviewXStoreModule = {
   state: () => ({
     config: {
+      debounceInMs: 250,
```

Comment:
are we actually using this? If we do, I do not see where 😅

Reply:
Nope, leftover from previous implementation (like 20 commits before 😆)